draft-ietf-tls-sni-encryption.md

```diff
@@ -450,6 +456,10 @@ In particular, the requirement to not stick out presented in
 (#snireqdontstickout) may have to be lifted, especially
 for proposed solutions that could quickly reach large scale deployments.
 
+The document mentions that encrypting SNI will simultaneously thwart invasions
+of the TLS exchange whose purpose is to improve some forms of security,
+and outlines the need of alternative solutions.
```

Comment:
We do discuss this topic elsewhere in the document, so it's not out of place to mention it again here.  But the rhetoric does not flow very well, with "simultaneously" only mentioning one thing it does and not also the (desired) primary purpose to balance it.  There's also not much of a transition from the previous paragraph, though in this style of writing there may not need to be one.

Reply:
Yes. I copied Mike's sentence, but it might be better to just point to the section that discusses the issue.

Reply:
Please check the new version.